tests/Feature/Sensors/ScheduledTaskSensorTest.php

```diff
@@ -125,8 +125,11 @@
 
 it('ingests failed tasks', function () {
     $line = __LINE__ + 1;
-    $task = app(Schedule::class)->call(fn () => travelTo(now()->addMicroseconds(1_000_000)) & throw new Exception('Unhandled error'))
-        ->everyMinute();
+    $task = app(Schedule::class)->call(function () {
+        travelTo(now()->addMicroseconds(1_000_000));
+
+        throw new Exception('Unhandled error');
+    })->everyMinute();
```

Comment:
This was previously using the bitwise `&` operator. Felt strange. Moved to a multiline.